gpuselect/__init__.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import print_function
 import os
```

Comment:
Because you're into flake8 and formatting, I'll mention that I think there should be a newline here.